recipes/genomics/Danio_rerio/danRer10/danrer10-gtf-refseq-ucsc-v1/meta.yaml

```diff
@@ -0,0 +1,39 @@
+build:
+  binary_relocation: false
+  detect_binary_files_with_prefix: false
+  noarch: generic
+  number: 0
+extra:
+  authors: yliu
+  extra-files: []
+package:
+  name: danrer10-gtf-refseq-ucsc-v1
+  version: '1'
+requirements:
+  build:
+  - gsort
+  - htslib
+  - zlib
+  run:
+  - gsort
+  - htslib
+  - zlib
+source:
+  path: .
+about:
+  identifiers:
+    genome-build: danRer10
+    species: Danio_rerio
+  keywords:
+  - GTF
+  - ReFSeq
+  - potein-coding
+  - non-protein-coding
+  - refGene
+  - UCSC
+  summary: 'The UCSC RefSeq gene track converted into GTF format containing known
+    zebrafish protein-coding and non-protein-coding genes from RefSeq. (RefSeq: NCBI
+    RNA reference sequence collection) (UCSC GTF format info: https://genome.ucsc.edu/FAQ/FAQformat.html#format3)'
+  tags:
+    data-version: Sept-2014
```

Comment:
That data version you have entered seems to be inconsistent with what is on the UCSC Genome Browser download website. Please ensure that the data version is correct. (The `--data_version` flag when using `ggd make-recipe`)